docs/tutorials/cluster.md

```diff
@@ -361,17 +361,17 @@ The resulting configs after the split:
 New Historical (on 2 Data servers)
 
 ```
- druid.processing.buffer.sizeBytes=500000000
- druid.processing.numMergeBuffers=8
- druid.processing.numThreads=31
+druid.processing.buffer.sizeBytes=500MiB
```

Comment:
Here too, if reverting the change above, along with line 374 below

Reply:
All configuration files are using `500MiB` or `100MiB` too.